tile-service/src/main/java/com/oculusinfo/tile/rendering/ImageRendererFactory.java

```diff
@@ -90,11 +90,6 @@ protected TileDataImageRenderer create () {
 			return new TopAndBottomTextScoresImageRenderer();
 		} else if ("doublestatistics".equals(rendererType)) {
 			return new DoublesStatisticImageRenderer();
-		} else if ("metadata".equals(rendererType)) {
-		    List<String> components = getPropertyValue(METADATA_COMPONENTS);
```

Comment:
The three properties used by the MetaDataRenderer are no longer needed; they should be removed too.